src/app/components/Validator/Validator.tsx

```diff
@@ -6,6 +6,7 @@ import { useParams } from 'react-router-dom';
 import Typography from '@material-ui/core/Typography';
 import config from '~app/common/config';
 import SsvNetwork from '~lib/api/SsvNetwork';
+import IsValid from '~app/common/components/IsValid/IsValid';
```

Comment:
This import should be done from `index.ts` instead of target file in that folder:

```
import IsValidBadge from '~app/common/components/IsValidBadge';
```